test/integration/functional_test.go

```diff
@@ -361,6 +361,23 @@ func validateExtraConfig(ctx context.Context, t *testing.T, profile string) {
 
 }
 
+func imageID(image string) string {
```

Comment:
add a comment what this func is for and why...

Reply:
done